.vscode/launch.json

```diff
@@ -5,11 +5,12 @@
     "version": "0.2.0",
     "configurations": [
         {
-            "type": "chrome",
+            "name": "Launch Chrome",
```

Comment:
Again - this file should not be in included in git commits. You need to list it in `.gitignore` file - https://git-scm.com/docs/gitignore